telegram/ext/conversationhandler.py

```diff
@@ -487,7 +487,7 @@ def check_update(self, update: HandlerArg) -> CheckUpdateType:  # pylint: disabl
 
     def handle_update(  # type: ignore[override]
         self,
-        update: HandlerArg,
+        update: Update,
```

Comment:
https://github.com/python-telegram-bot/python-telegram-bot/blob/1ddffc2f6321218fd5ec03af6bf3187de565deb0/telegram/ext/conversationhandler.py#L500

why is this still only a telegram update?

Reply:
`check_update` must accept `Any`, `handle_update` needs only accept what `check_update` allows, so `tg.Update` for all the TG-handlers

Reply:
ahhh I see because we only allow telegram updates in check update, right.